packages/devtools_app/lib/src/debugger/debugger_state.dart

```diff
@@ -84,11 +84,11 @@ class DebuggerState {
     }
   }
 
-  Future<Success> pause() => _service.pause(isolateRef.id);
```

Comment:
why are these types removed? Were they incorrect?

Reply:
yes vm_service 3.0.0 included a breaking change. The type is now dynamic as a Sentinel can be returned

Reply:
Any RPC that requires an isolateId parameter can return a Sentinel if the isolate no longer exists. The spec didn't explicitly state this even though this has always been true, so it was possible to run into runtime type errors if timing was bad.